.github/workflows/release.yml

```diff
@@ -0,0 +1,144 @@
+          security unlock-keychain -p ${{ env.KEYCHAIN_PASSWORD }} ${{ env.KEYCHAIN }}
+          security import ${{ env.INSTALLER_CERT_MAC_PATH }} -k ${{ env.KEYCHAIN }} -f pkcs12 -A -T /usr/bin/codesign -P "${{ secrets.INSTALLER_CERT_MAC_PASSWORD }}"
+          security set-key-partition-list -S apple-tool:,apple: -s -k ${{ env.KEYCHAIN_PASSWORD }} ${{ env.KEYCHAIN }}
+
+      - name: Install gon for code signing and app notarization
+        run: |
+          wget -q https://github.com/mitchellh/gon/releases/download/v0.2.3/gon_macos.zip
+          unzip gon_macos.zip -d /usr/local/bin
+
+      - name: Sign and notarize binary
+        env:
+          AC_USERNAME: ${{ secrets.AC_USERNAME }}
+          AC_PASSWORD: ${{ secrets.AC_PASSWORD }}
+        run: |
+          gon gon.config.hcl
+
+      - name: Re-package binary and update checksum
+        # Repackage the signed binary replaced in place by Gon (ignoring the output zip file)
+        run: |
+          TAG=${GITHUB_REF/refs\/tags\//}
+          tar cjf dist/FirmwareUpdater_${TAG}_macOS_64bit.tar.bz2 \
+          firmwares \
+          LICENSE.txt \
+          -C dist/macos64/ FirmwareUpdater
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: dist
+
+  create-release:
+    runs-on: ubuntu-latest
+    needs: notarize-macos
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: dist
+          # to ensure compatibility with v1
+          # https://github.com/actions/download-artifact#compatibility-between-v1-and-v2
+          path: dist
+
+      - name: Install Taskfile
+        uses: arduino/actions/setup-taskfile@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          version: 3.x
+
+      - name: Build
+        # This must be done after OS X notarization or the wrong checksum
+        # would be calculated since the binary is modified during notarization
+        run: task dist:generate-index-data
+
+      - name: Identify Prerelease
+        # This is a workaround while waiting for create-release action
+        # to implement auto pre-release based on tag
+        id: prerelease
+        run: |
+          wget -q -P /tmp https://github.com/fsaintjacques/semver-tool/archive/3.0.0.zip
+          unzip -p /tmp/3.0.0.zip semver-tool-3.0.0/src/semver >/tmp/semver && chmod +x /tmp/semver
+          if [[ $(/tmp/semver get prerel ${GITHUB_REF/refs\/tags\//}) ]]; then echo "::set-output name=IS_PRE::true"; fi
+
+      - name: Create Github Release and upload artifacts
+        uses: ncipollo/release-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          bodyFile: "dist/CHANGELOG.md"
+          draft: false
+          prerelease: ${{ steps.prerelease.outputs.IS_PRE }}
+          artifacts: dist/FirmwareUpdater*
```

Comment:
This causes `package_index.json` to not be added as a release artifact (https://github.com/per1234/FirmwareUpdater/releases/tag/0.53.1). Not knowing how that file is to be consumed, I can't say whether it's an issue or not, so just pointing it out.

Reply:
Thanks, I forgot to add it.